include/lineview.h

```diff
@@ -0,0 +1,27 @@
+#ifndef NEWSBOAT_LINEVIEW_H_
+#define NEWSBOAT_LINEVIEW_H_
+
+#include <string>
+
+#include "stflpp.h"
+
+namespace newsboat {
+
+class LineView {
+public:
+	LineView(Stfl::Form& form, const std::string& name);
+
+	void setText(const std::string& text);
+	std::string getText();
```

Comment:
I might be wrong, but wouldn't methods (in new code) use snake_case?

Reply:
Jup, completely right.
I just keep forgetting because I got so used to a style used at work.

Reply:
I hear ya! ;-)